server/server.js

```diff
@@ -77,6 +72,40 @@ app.get('/api/products/:productId', async (req, res, next) => {
   }
 });
 
+// GET user's account info
+app.get('/api/tables/public.users', authMiddleware, async (req, res, next) => {
```

Comment:
It looks like this returns _all_ the users in your database. When will you use that? I would think that instead you want to return the info for a single user. For that, you would want something like `/api/tables/users/:userId` and you would add a where clause in your query `where "userId" = $1` and pass `userId` as the first parameter.

Reply:
Got it!